<?xml version="1.0" encoding="utf-8"?>

<launch>

    <include file="$(find drone_controller)/launch/main_mixedreality.launch" />

    <node pkg="drone_controller" name="program_controller_node" type="gate_scenario_controller.py" output="screen">
    </node>

    <!-- Error is a value between -1 an 1, and the output is a value between -100, 100 -->
    <node pkg="drone_controller" name="gate_navigation_node" type="gate_navigation.py" output="screen">
        <param name="gains/p" type="double" value="25" />
        <param name="gains/i" type="double" value="0" />
        <param name="gains/d" type="double" value="15" />
    </node> 


    <node pkg="rqt_image_view" type="rqt_image_view" name="gate_center" args="/gate_navigation/sensors/camera" output="screen"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="mixer_img" args="/mixer/sensors/camera" output="screen"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="test_color" args="/uav1/overlay/camera" output="screen"/>
</launch>

